internal/schemautil/schemautil.go

```diff
@@ -201,6 +201,10 @@ func IpFilterValueDiffSuppressFunc(k, old, new string, _ *schema.ResourceData) b
 	return old == "0.0.0.0/0" && new == "" && strings.HasSuffix(k, ".ip_filter.0")
 }
 
+func TrimSpaceDiffSuppressFunc(k, old, new string, _ *schema.ResourceData) bool {
+	return strings.TrimSpace(old) == strings.TrimSpace(new)
```

Comment:
is this really necessary?

Reply:
Yes. Backend returns certificate with trailing \n.

Reply:
why can't we have it with the trailing `\n`?

Reply:
Meaning? it's compared with local one and doesn't match.  
So either you add trailing to one or trim it from another :)

Reply:
I'm sorry but according to [RFC 7468 ("Textual Encodings of PKIX, PKCS, and CMS Structures", section 3: ABNF)](https://rfc-editor.org/rfc/rfc7468#section-3), we can't trim the trailing whitespace at the very end of the string, could you please confirm that only the `\n` is being trimmed?

Reply:
`strings.TrimSpace` trims `\n`, I guess we should use https://golang.org/pkg/strings/#TrimLeft instead?

Reply:
after discussing this internally, this has been concluded to be left as is

Reply:
The certificate is encapsulated between `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----`. Terraform's multiline markup is quite tricky and produces unwanted whitespaces https://www.terraform.io/language/expressions/strings#heredoc-strings. It's more user friendly just trim that out. Certificate won't be damaged. A real-world ACC test example with server side validation is attached to the PR.